trivia.js

```diff
@@ -83,28 +76,36 @@ triviaGame.start()
 
     //need to add logic to randomize answers
     for(i=0; i<triviaGame.questions.length; i++){
-      eval('var question'+ i + ' = new Question(triviaGame.incorrect_answers[i],triviaGame.correct_answers[i],triviaGame.questions[i])');
-      $("#question"+i+"question_display").html(triviaGame.questions[i]);
-      $("#question"+i+"answerA_display").html(triviaGame.correct_answers[i]);
-      $("#question"+i+"answerB_display").html(triviaGame.incorrect_answers[i][0]);
-      $("#question"+i+"answerC_display").html(triviaGame.incorrect_answers[i][1]);
-      $("#question"+i+"answerD_display").html(triviaGame.incorrect_answers[i][2]);
-
-    } 
+      var q = triviaGame.questions[i];
+      $(`#question${i}question_display`).html(q.question);
+
+      $("#question"+i+"answerA_display").html(q.correctAns);
+      $(`#ans${i}A`).val(q.correctAns);
```

Comment:
Used values because when you store stuff as HTML the text can get converted. So HTML for display and Values for data storage.